server/modules/slack/slackIntegration.js

```diff
@@ -160,3 +169,34 @@ export const addToChannel = async (fellowEmail, channel) => addOrRemove(fellowEm
  * @returns {Object} - The result after a user has been removed from a channel
  */
 export const removeFromChannel = async (fellowEmail, channel) => addOrRemove(fellowEmail, channel, 'kick');
+
+/**
+ * @function onboardingSlackAutomation
+ * @desc combines all onboarding functions
+ *       into one function.
+ *
+ * @param {string} fellowEmail - user's email
+ * @param {string} removalChannelId - the id of the channel you are removing the developer
+ * @param {string} partnerId - the partner id
+ *
+ * @returns {Object} - With a success message after slack automation has completed
+ */
+
+export const onboardingSlackAutomation = async (fellowEmail, channel, partnerId) => {
```

Comment:
There is kind of a mismatch between the parameters this function is receiving and the JSDoc comment. 
The JSDoc has `removalChannelId` while the function itself is taking `channel`.
Note that whatever you specify in the JSDoc as parameters(`@param`) must correlate with what the function is actually taking as parameters.

Reply:
Awesome!